### PR TITLE
update description in ADD EDGE NODES

### PR DESCRIPTION
**Pull request by Bettygogo2021**
Signed-off-by: Bettygogo2021 <91529409+Bettygogo2021@users.noreply.github.com>
Hi, folks, I have updated the docs as suggested, pls help me review this PR. 3ks.
/assign @benjaminhuo @wenchajun

### CODE REVIEW

**Comment by Bettygogo2021** [comment]
I've updated the docs. For suggestions, pls leave a comment.

**Comment by Bettygogo2021** [comment]
@benjaminhuo Ben, I've updated the docs. Pls check it.

File: content/en/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md [hunk 1]
```diff
@@ -24,9 +24,35 @@ This tutorial demonstrates how to add an edge node to your cluster.
 - You have an available node to serve as an edge node. The node can run either Ubuntu (recommended) or CentOS. This tutorial uses Ubuntu 18.04 as an example.
 - Edge nodes, unlike Kubernetes cluster nodes, should work in a separate network.
 
+## Evict non-edge workloads
```
Comment (benjaminhuo) [inline content/en/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md:27]:
```suggestion
## Prevent non-edge workloads from being scheduled to edge nodes
```

File: content/en/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md [hunk 2]
```diff
@@ -24,9 +24,35 @@ This tutorial demonstrates how to add an edge node to your cluster.
...
+for((i=0;i<length;i++));  
+do
+         ds=${DaemonSets[$i]}
+        echo "Patching resources:DaemonSet/${ds}" in ns:"$ns",
+        kubectl -n $ns patch DaemonSet/${ds} --type merge --patch "$NoShedulePatchJson"
+        sleep 1
+done
+```
+
 ## Configure an Edge Node
```
Comment (benjaminhuo) [inline content/en/docs/v3.3/installing-on-linux/cluster-operation/add-edge-nodes.md:53]: `## Configure an Edge Node` should be moved to a position between `## Create Firewall Rules and Port Forwarding Rules` and `## Add an Edge Node`

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Comment by Bettygogo2021** [comment]
@benjaminhuo Ben, pls check the updates.

**Comment by Bettygogo2021** [comment]
@benjaminhuo Ben, already update as suggested. Pls tag /lgtm if the updates are as expected. 3ks for your suggestions.

**Pull request merged**